doc/plugin_server_bundlepublisher_aws_rolesanywhere_trustanchor.md

```diff
@@ -0,0 +1,32 @@
+# Server plugin: BundlePublisher "aws_rolesanywhere_trustanchor"
+
+**Note: This plugin is only supported when an UpstreamAuthority plugin is used.**
```

Comment:
I think that it may be a good idea to use a Warning note highlighted by markdown: 
```suggestion
> [!WARNING]
> This plugin is only supported when an UpstreamAuthority plugin is used.
```